game/scripts/vscripts/heroes/vengefulspirit/command_aura.lua

```diff
@@ -0,0 +1,165 @@
+---@class vengefulspirit_command_aura_lua : CDOTA_Ability_Lua
+vengefulspirit_command_aura_lua = class({})
```

Comment:
Would like to do this without _lua, but I'm pretty sure that causes some weird engine side-effects. I think we just enforce *_lua naming convention.